### PR TITLE
[4.0] Fix Debug's Session output

### PR DESCRIPTION
**Pull request by Sophist-UK**
### Summary of Changes

Fix non-output of session variables in debug output.

### Testing Instructions

Before application of this PR, open up the Session section of debug output and there will be no information except the main subheadings.

Apply this PR and session values will be shown.

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ef0c9873b406461efbfb110523137ea01d578b76<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19378">issues.joomla.org/tracker/joomla-cms/19378</a>.</sub>

**Comment by Sophist-UK** [comment]
There is still potentially a CSS issue. If the `<pre>` item for an individual session variable is wider than the screen in responsive mode, the the individual `<pre>` block gets a horizontal scroll bar. The alternatives are:

a. Leave it as is - its not very pretty but it is functional.
b. Scroll the entire subsection - in which case names go off the screen.
c. Wrap the text in the pre tag - which I think is the best approach.

I have added a commit for this to this PR.

**Comment by Sophist-UK** [comment]
Discovered a duplicate CSS file so have updated that too.

File: media/plg_system_debug/css/debug.css [hunk 1]
```diff
@@ -189,6 +189,13 @@ div#system-debug {
 	border: 0;
 	margin: 0;
 	padding: 0;
+	white-space: pre-wrap;       /* css-3 */
+	white-space: -moz-pre-wrap !important;  /* Mozilla, since 1999 */
+	white-space: -pre-wrap;      /* Opera 4-6 */
+	white-space: -o-pre-wrap;    /* Opera 7 */
+	white-space: -webkit-pre-wrap; /* Newer versions of Chrome/Safari*/
+	word-wrap: break-word;       /* Internet Explorer 5.5+ */
+	word-break: break-word;
```
Comment (C-Lodder) [inline media/plg_system_debug/css/debug.css]: We don't support older browsers, please just stick with modern support:

```
white-space: pre-wrap;
```

You can remove the other lines you added above

File: media/system/css/debug.css [hunk 2]
```diff
@@ -189,6 +189,13 @@ div#system-debug {
 	border: 0;
 	margin: 0;
 	padding: 0;
+	white-space: pre-wrap;       /* css-3 */
+	white-space: -moz-pre-wrap !important;  /* Mozilla, since 1999 */
+	white-space: -pre-wrap;      /* Opera 4-6 */
+	white-space: -o-pre-wrap;    /* Opera 7 */
+	white-space: -webkit-pre-wrap; /* Newer versions of Chrome/Safari*/
+	word-wrap: break-word;       /* Internet Explorer 5.5+ */
+	word-break: break-word;
```
Comment (C-Lodder) [inline media/system/css/debug.css]: same applies here

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 09cd5f456db46afaf01421e0e3ac23c227dce6b4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19378">issues.joomla.org/tracker/joomla-cms/19378</a>.</sub>

**Comment by ghost** [comment]
@brianteeman can you please retest?

**Comment by wilsonge** [comment]
Thankyou!

**Pull request merged**